lib/http-bridge/test/integration/Main.hs

```diff
@@ -71,13 +77,17 @@ main = hspec $ do
     describe "Cardano.WalletSpec" Wallet.spec
     describe "Cardano.Wallet.HttpBridge.NetworkSpec" HttpBridge.spec
     describe "CLI commands not requiring bridge" MnemonicsCLI.spec
-    beforeAll startCluster $ afterAll killCluster $ after tearDown $ do
+    beforeAll (startCluster ListenOnRandomPort) $ afterAll killCluster $ after tearDown $ do
```

Comment:
This shouldn't be a problem for the API spec as long as we sort out the baseUrl in the manager, but the CLI specs also need to know about the port; So we probably gotta add it to the `Context` I believe.

Reply:
I change it from `IO ()` to `IO Warp.Port` in start Server in `Cardano.Wallet.Api.Server` now we need to accommodate it in integration tests